POEM_096/POEM_096.md

```diff
@@ -0,0 +1,34 @@
+POEM ID:  096
+
+Title:  Research and Pseudocode of Manual Reverse Mode differentiation
+
+authors: jsrogan, coleyoung5
+
+Competing POEMs:  N/A
+
+Related POEMs: N/A
+
+Associated implementation PR: N/A
+
+
+Status:
+
+- [x] Active
+- [ ] Requesting decision
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Integrated
+
+
+
+## Motivation
+Considering the fact that private optimization libraries such as SNOPT offer the ability to find a feasible starting point, it makes sense that OPENMdao should also have that functionality. Aditionally OPENMdao's current components do not calculate reverse derivatives, so the implementation of this is also necessary (and still ongoing).
```

Comment:
OpenMDAO computes partials at the component level, either through specification of a jacobian matrix - or through a "matrix-free" API (the component.compute_jacvec_prod method) in it will compute a forward JVP or a reverse VJP. The framework then assembles these calculations into a graph for efficient computation and assembly of the total derivatives. The notion of reverse derivatives just aren't relevant to the implementation of a feasible starting point.